components/expenses/__tests__/Expenses.test.js

```diff
@@ -80,7 +80,7 @@ describe('Expenses component', () => {
       expect(component.find('[data-cy="mark-expense-as-paid-btn"]')).not.toBeFalsy();
 
       // make sure none are disabled
-      expect(component.find('[data-cy="expense-actions"] button[disabled]').lenght).toEqual(undefined);
+      expect(component.find('[data-cy="expense-actions"] button[disabled]').length).toEqual(undefined);
```

Comment:
> Also, one test is impacted by the change, could you look into it?

this test is failing, but I think it should be `length`. So I guess code should be reviewed.

Reply:
Yes, I agree, better remove the change and create a separated Issue or PR.